src/test/ui/async-await/issues/issue-78938-async-block.stderr

```diff
@@ -0,0 +1,30 @@
+error[E0373]: async block may outlive the current function, but it borrows `room_ref`, which is owned by the current function
+  --> $DIR/issue-78938-async-block.rs:8:39
+   |
+LL |       let gameloop_handle = spawn(async {
+   |  _______________________________________^
+LL | |         game_loop(Arc::clone(&room_ref))
+   | |                               -------- `room_ref` is borrowed here
+LL | |     });
+   | |_____^ may outlive borrowed value `room_ref`
+   |
+   = note: borrows cannot be held across a yield point, because the stack space of the current function is not preserved
+   = help: see https://rust-lang.github.io/async-book/03_async_await/01_chapter.html#awaiting-on-a-multithreaded-executor for more information
+note: function requires argument type to outlive `'static`
```

Comment:
Can you make these part of the same note? This is specific to executors that require their argument to be `'static`, it doesn't happen for normal async blocks.

Reply:
Sorry, I'm unsure of what you mean. Did you mean making the note: and help: messages I added to be part of "function requires ..." error instead of the "async block ..." error?

Reply:
Yes, that's what I meant.

Reply:
My new commit makes those note: and help: messages part of the "function requires..." error